lib/managers/webhooks.js

```diff
@@ -0,0 +1,145 @@
+/**
+* @fileoverview Webhook for the Box Webhooks Resource
+*/
+'use strict';
+
+// ------------------------------------------------------------------------------
+// Requirements
+// ------------------------------------------------------------------------------
+
+var urlPath = require('../util/url-path');
+
+// ------------------------------------------------------------------------------
+// Private
+// ------------------------------------------------------------------------------
+
+// Base path for all webhooks endpoints
+var BASE_PATH = '/webhooks';
+
+// ------------------------------------------------------------------------------
+// Public
+// ------------------------------------------------------------------------------
+
+/**
+* Simple manager for interacting with all 'Webhooks' endpoints and actions.
+*
+* @param {BoxClient} client The Box API Client that is responsible for making calls to the API
+* @constructor
+*/
+function Webhooks(client) {
+	// Attach the client, for making API calls
+	this.client = client;
+}
+
+/**
+* If an event trigger is successfully attached to a file or folder then a JSON object representing the
+* webhook is returned to you.
+*
+* API Endpoint: '/webhooks'
+* Method: POST
+*
+* @param {string} targetID - File or Folder ID to create webhook on
+* @param {string} targetType - Value of FILE or FOLDER
+* @param {string} notificationURL - The url of your application where Box will notify you of events triggers
+* @param {string[]} triggerTypes - Array of event types that trigger notification for the target
```

Comment:
should this be" 

@param {WebhookTriggerType} 

or 

@param {WebhookTriggerType[]}